### DISCUSSION TITLE
(vlc.portable) Package only installs 32-bit version

### DISCUSSION THREAD

**Discussion by kwencel** [chocolatey-community/chocolatey-packages#1991] [Ideas]
This package always installs 32-bit version while 64-bit is also available. I believe it should check the system architecture and install 64-bit version if it can.

**Comment by RedBaron2** [comment]
@kwencel 
Currently this package only checks for the 32bit version. If you are able to make the changes needed to update this package. I'm sure that the PR would be helpful for others.
